2013/09/04 16:24:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/index.php/gii
---
2013/09/04 17:00:47 [error] [exception.CException] exception 'CException' with message 'Property "User.last_login" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('last_login')
#1 C:\xampp\htdocs\vimm\protected\components\UserIdentity.php(30): CActiveRecord->__get('last_login')
#2 C:\xampp\htdocs\vimm\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vimm/index.php/site/login
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/login
---
2013/09/04 17:04:35 [error] [exception.CException] exception 'CException' with message 'Property "User.last_login" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('last_login', 1378307075)
#1 C:\xampp\htdocs\vimm\protected\components\UserIdentity.php(30): CActiveRecord->__set('last_login', 1378307075)
#2 C:\xampp\htdocs\vimm\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vimm/index.php/site/login
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/login
---
2013/09/04 17:08:42 [error] [exception.CException] exception 'CException' with message 'Property "User.last_login" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('last_login', 1378307322)
#1 C:\xampp\htdocs\vimm\protected\components\UserIdentity.php(30): CActiveRecord->__set('last_login', 1378307322)
#2 C:\xampp\htdocs\vimm\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vimm/index.php/site/login
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/login
---
2013/09/04 17:08:55 [error] [exception.CException] exception 'CException' with message 'Property "User.last_login" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('last_login', 1378307335)
#1 C:\xampp\htdocs\vimm\protected\components\UserIdentity.php(30): CActiveRecord->__set('last_login', 1378307335)
#2 C:\xampp\htdocs\vimm\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vimm/index.php/site/login
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/login
---
2013/09/04 17:09:04 [error] [exception.CException] exception 'CException' with message 'Property "User.last_login" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('last_login', 1378307344)
#1 C:\xampp\htdocs\vimm\protected\components\UserIdentity.php(30): CActiveRecord->__set('last_login', 1378307344)
#2 C:\xampp\htdocs\vimm\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vimm/index.php/site/login
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/login
---
2013/09/04 17:31:21 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "register".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#1 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(148): CController->render('register', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php
---
2013/09/04 17:31:54 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "register".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#1 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(148): CController->render('register', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php
---
2013/09/04 18:06:56 [error] [exception.CException] exception 'CException' with message 'Property "User.activation" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('activation', '5e36238dcd5439e...')
#1 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(123): CActiveRecord->__set('activation', '5e36238dcd5439e...')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/register
---
2013/09/04 18:52:44 [error] [exception.CException] exception 'CException' with message 'Property "User.password_repeat" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password_repeat')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('password_repeat')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(User), 'password_repeat')
#3 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(User), 'password_repeat', Array)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(User), 'password_repeat', Array)
#5 C:\xampp\htdocs\vimm\protected\views\site\register.php(42): CActiveForm->textField(Object(User), 'password_repeat')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#10 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(134): CController->render('register', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/register
---
2013/09/04 18:53:01 [error] [exception.CException] exception 'CException' with message 'Property "User.password_repeat" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password_repeat')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('password_repeat')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(User), 'password_repeat')
#3 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(User), 'password_repeat', Array)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(User), 'password_repeat', Array)
#5 C:\xampp\htdocs\vimm\protected\views\site\register.php(42): CActiveForm->textField(Object(User), 'password_repeat')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#10 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(134): CController->render('register', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/register
---
2013/09/04 18:53:28 [error] [exception.CException] exception 'CException' with message 'Property "User.password_repeat" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password_repeat')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('password_repeat')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(User), 'password_repeat')
#3 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(User), 'password_repeat', Array)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(User), 'password_repeat', Array)
#5 C:\xampp\htdocs\vimm\protected\views\site\register.php(42): CActiveForm->textField(Object(User), 'password_repeat')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#10 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(134): CController->render('register', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/register
---
2013/09/04 18:53:32 [error] [exception.CException] exception 'CException' with message 'Property "User.password_repeat" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('password_repeat')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('password_repeat')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(User), 'password_repeat')
#3 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(User), 'password_repeat', Array)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(User), 'password_repeat', Array)
#5 C:\xampp\htdocs\vimm\protected\views\site\register.php(42): CActiveForm->textField(Object(User), 'password_repeat')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('register', Array, true)
#10 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(134): CController->render('register', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('register')
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/register')
#17 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/vimm/index.php/site/register
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/register
---
2013/09/04 18:54:16 [error] [php] Missing argument 2 for CActiveForm::labelEx(), called in C:\xampp\htdocs\vimm\protected\views\site\register.php on line 41 and defined (C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php:595)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 C:\xampp\htdocs\vimm\protected\controllers\SiteController.php(134): SiteController->render()
#4 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionRegister()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#11 C:\xampp\htdocs\vimm\index.php(13): CWebApplication->run()
REQUEST_URI=/vimm/index.php/site/register
in C:\xampp\htdocs\vimm\protected\views\site\register.php (41)
in C:\xampp\htdocs\vimm\protected\controllers\SiteController.php (134)
in C:\xampp\htdocs\vimm\index.php (13)
2013/09/04 21:39:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "mapa".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('mapa')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('mapa')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/mapa')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/mapa
HTTP_REFERER=http://127.0.0.1/vimm/index.php
---
2013/09/04 21:43:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "mapa".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('mapa')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('mapa')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/mapa')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/mapa
HTTP_REFERER=http://127.0.0.1/vimm/index.php
---
2013/09/04 21:46:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "mapa".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('mapa')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('mapa')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/mapa')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/mapa
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/index
---
2013/09/04 21:46:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "mapa".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('mapa')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('mapa')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/mapa')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/mapa
HTTP_REFERER=http://127.0.0.1/vimm/index.php/site/index
---
2013/09/04 22:43:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "mapa".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('mapa')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('mapa')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/mapa')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vimm\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/vimm/index.php/site/mapa
HTTP_REFERER=http://127.0.0.1/vimm/index.php
---
